src/web/mjs/connectors/MangaHost.mjs

```diff
@@ -9,6 +9,17 @@ export default class MangaHost extends Connector {
         super.label = 'MangaHost';
         this.tags = ['manga', 'portuguese'];
         this.url = 'https://mangahostz.com';
+
+        this.config = {
+            throttle: {
```

Comment:
The magic key `throttle` will also be used by the downloader to delay image downloads by the given value. If not intended the property shoild be renamed to something else.